packages/api/src/users/users.mapper.ts

```diff
@@ -14,13 +14,13 @@ export class UsersMapper {
 
   static fromJwtToDomain(payload: JwtPayload): UserFromJwt {
```

Comment:
I'd recommend to get rid of those mappers.

Reply:
Agree, but looks like an out-of-scope task from this PR perspective